activemq/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG - activemq
 
+## 1.4.0 / 2020-01-13
+
+* [Added] Add test environment for activemq. See [#5204](https://github.com/DataDog/integrations-core/pull/5204).
```

Comment:
Should be a `no-changelog`

Reply:
@florimondmanca I changed the label on https://github.com/DataDog/integrations-core/pull/5204

For this PR, we can just remove the changes related to `activemq`.

Reply:
Yes, I'll revert the commit and run `ddev release make all --sign-only` to update the release metadata.